README.md

```diff
@@ -672,6 +672,38 @@ comparison operations are not instrumented and default backends do not make use
 this feature. If your backend requires instrumenting floating point comparisons, you
 must call `verificarlo` with the `--inst-fcmp` flag.
 
+## Precision customisation
+
+When the `--inst-func` flag is given, verificarlo instrument function's calls and 
+allow the user to specify the precision used for each function.
+
+To do it, set a new environement variable with the name of an output file:
```

Comment:
Before giving the details on the file formats; we should explain what is Precision Customization; and what problem it solves.